src/auth/token-generator.ts

```diff
@@ -213,6 +226,36 @@ export class IAMSigner implements CryptoSigner {
       );
     });
   }
+
+  /**
+   * @inheritdoc
+   */
+  public getAlgorithm(): string {
+    return ALGORITHM_RS256;
+  }
+}
+
+/**
+ * A CryptoSigner implementation that is used when communicating with the Auth emulator.
+ * It produces unsigned tokens.
+ */
+export class EmulatedSigner implements CryptoSigner {
+
+  algorithm = ALGORITHM_NONE;
+
+  /**
+   * @inheritDoc
+   */
+  public sign(_: Buffer): Promise<Buffer> {
+    return Promise.resolve(Buffer.from(''));
+  }
+
+  /**
+   * @inheritDoc
+   */
+  public getAccountId(): Promise<string> {
+    return Promise.resolve('firebase-auth-emulator@example.com');
```

Comment:
I'm not so sure about this ... could we do something better here?

Reply:
cc @yuchenshi

Reply:
This is probably the best thing we can do